docs/BUILDING_CYGWIN.md

```diff
@@ -1,4 +1,17 @@
-# Building mips-linux-binutils on Windows using Cygwin
+# Building with Cygwin
+
+**N.B.** These have not been tested recently, you may find the requirements have changed a bit.
```

Comment:
I agree with the other other comment, but this one I feel makes more sense with "not [...] recently", in that if someone does test them, hopefully they will update the docs and alter this comment appropriately. I think "not [...] recently" is useful here firstly because it will remain true "by default" (unlike the "currently" one), and secondly because it conveys that at some point they did work, but you shouldn't necessarily expect it to work "out-of-the-box" now.

Reply:
You could reword it as something like "not been tested since spring 2021" (or whenever idk)
Like on Wikipedia, it's bad practice to use relative time for something that may last a "long" time so they encourage refering to actual dates

Reply:
I think this is good in principle, but in this specific case i honestly have no idea when cygwin was last tested lol
when it was added to the repo maybe?

I dont have an opinion much here either way. I think "recently" here is more of a general statement and not so much an indicator of a specific time. at least not in a way that would be confusing. but yeah if theres a better way to word it then sure

Reply:
I understand the point, but I don't think it's actually that helpful to be more specific: it indicates that something may be broken, but whether it's a little time or a long time usually doesn't have a bearing on how difficult it might be to get it to work.